content/api/rest-api.textile

```diff
@@ -675,6 +675,32 @@ bc[json]. [{
   extras: <optional extras object>,
 }]
 
+h3(#annotations-client-summary). Retrieve annotation summary for a specific clientId
+
+h6. GET rest.ably.io/channels/@<channelId>@/messages/@<messageSerial>@/client-summary
```

Comment:
The base branch likely needs rebasing as well, but I recently updated all endpoints to the new format (`main.realtime.ably.net`)